html/docs/include/features.php

```diff
@@ -907,6 +907,14 @@
 		example find out how good your unit tests are.',
 		""
 	),
+    'garbage_collection' => array(
+        'Garbage Collection',
+        FUNC_GARBAGE_COLLECTION,
+        'Xdebug\'s built-in garbage collection statistics profiler allows you to
+        find out when the PHP internal garbage collector triggers, how many variables
+        it was able to clean up, how long it took and how how much memory was actually freed.',
+        ""
+    ),
```

Comment:
Tabs, not spaces, please.

Reply:
It would also be great to add an example of how the file looks like, and what the meaning of all the files are. The documentation in ``features.php`` is supposed to explain people how things work, a little like a tutorial.